### ISSUE TITLE
Huge window controls after the latest update

### ISSUE THREAD

**Issue by Udara-U3** [tista500/Adapta#188]
![screenshot from 2016-08-02 11-22-37](https://cloud.githubusercontent.com/assets/12265797/17327781/0b544838-58cb-11e6-8edf-270de92f5ac4.png)

**Reporter info**

 ```
 * Distribution - Ubuntu 16.10 Yekety Yak
 * Gtk+ 3.0 version - 3.18
 * Desktop environment - Budgie Desktop
 ```

**Adapta version (if you knew)**

3.21.4 Stable Release (rev.116)

**Related Application and/or Shell theming (or name of widget)**

Gnome Terminal

**Actual issue**

After the last update, Window controls go huge size in Gnome Terminal, and disappeared Adapta effects from it.

**Comment by tista500** [comment]
??

I could not reproduce.
And Gtk+ 3.20.6 series had already landed in proposed repository, but you really run Gtk+ 3.18.9 series?
Gnome-Terminal wasn't updated after released in Apr...

Is it really theme-specific issue?

Regards.

**Comment by tista500** [comment]
I'm re-installing Yakkety and retry again later...

**Comment by Udara-U3** [comment]
@tista500 Just landed GTK 3.20 but I have same problem :grin: Also same problem with Inkscape and Firefox too. I don't have any idea what is the actual problem :confused:

**Comment by Udara-U3** [comment]
Ohh it was GTK problem :smile: Latest update fixed everything, Thanks for your support.
